### PR TITLE
[BUG] - Link Tile hyphens Safari (issue/3174)

### LINKED ISSUE #3174: [BUG] - Link Tile hyphens Safari
### Contact Details

_No response_

### JS Framework

NextJs

### Porsche Design System version

3.13.0

### On which device(s) you can see the bug?

_No response_

### In which browsers you can see the bug?

Safari

### Browser version

_No response_

### Reproduction Example

https://shop.porsche.com/de/de-DE/c/porsche-accessoires?hierarchicalMenu=hierarchicalCategories.lvl0%3A+Accessoires&price=10%3A137

### Steps to reproduce the behavior

1. Go to https://shop.porsche.com/de/de-DE/c/porsche-accessoires?hierarchicalMenu=hierarchicalCategories.lvl0%3A+Accessoires&price=10%3A137
2. Scroll down to link tile
3. resize browser 
or https://jira.porsche.codes/browse/PSL-2271

image

image

### Expected behavior

All elements should be positioned correctly

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by kaiszybiak**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/issue/3174/

#### Scope

The added CSS properties ensure consistent text behavior on both Chrome and Safari during resizing.

image

#### Resolved Issue

Resolves #3174

### CODE REVIEW

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0),
 - `PTextFieldWrapper`, `PTextareaWrapper`: conditionally rendered component throws
   `TypeError: Cannot read properties of undefined (reading 'type')`
   ([#3383](https://github.com/porsche-design-system/porsche-design-system/pull/3383))
+- `LinkTile`: broken word-break & hyphens Safari
```
Comment (stephanschroeter) [inline packages/components/CHANGELOG.md:24]:
```suggestion
- `LinkTile`: Broken word-break & hyphens in Safari
```

File: packages/components/src/styles/tile/button-link-tile-styles.ts [hunk 2]
```diff
@@ -39,6 +39,8 @@ export const getButtonLinkTileStyles = (
         margin: 0,
         ...textLargeStyle,
         hyphens: 'inherit',
+        overflowWrap: 'anywhere',
```
Comment (stephanschroeter) [inline packages/components/src/styles/tile/button-link-tile-styles.ts:42]: maybe add a comment to both css props why you have chosen them.

Comment (kaiszybiak) [reply packages/components/src/styles/tile/button-link-tile-styles.ts:48]: added comment

File: packages/components/CHANGELOG.md (hunk 1 above)
Comment (kaiszybiak) [reply packages/components/CHANGELOG.md]: updated

**Pull request merged**